user/languages/javascript-with-nodejs.md

```diff
@@ -121,6 +121,51 @@ npm install
 
 Note that `npm install` can fail if a shrinkwrapped git dependency pointing to a branch has its HEAD changed.
 
+## Ember Apps
+You can build your Ember Apps on Travis CI and can automate test. By default ember comes with [`Qunit`](http://qunitjs.com/) as testing framework
+
+```yaml
+dist: trusty
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+language: node_js
+node_js:
+  - "0.12"
+env:
+    - EMBER_VERSION=default
+    # - EMBER_VERSION=release
+    # - EMBER_VERSION=beta
+    # - EMBER_VERSION=canary
+matrix:
```

Comment:
we can run ember in different releases such as canary , beta , release etc. here we set the version as default which will set the latest stable version.